config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -38,6 +38,16 @@ spec:
           spec:
             description: NodeFeatureDiscoverySpec defines the desired state of NodeFeatureDiscovery
             properties:
+              customConfig:
+                description: ConfigMap describes configuration options for the NFD
```

Comment:
Not true, this will only contain rules for the custom source

Reply:
fixed